core-web/libs/portlets/edit-ema/portlet/src/lib/dot-ema-shell/components/edit-ema-navigation-bar/edit-ema-navigation-bar.component.html

```diff
@@ -38,7 +38,7 @@
             [ngTemplateOutletContext]="{ item: item }">
         </ng-container>
         <span class="item__label" data-testId="nav-bar-item-label">
-            {{ item.label }}
+            {{ item.label | dm }}
```

Comment:
This should make a test fail to avoid this.

Reply:
Updated and fixed the test!